connexion/decorators/validation.py

```diff
@@ -101,7 +101,8 @@ def wrapper(request):
             if all_json(self.consumes):
                 data = request.json
 
-                if data is None and len(request.body) > 0 and not self.is_null_value_valid:
+                empty_body = len(request.body) == 0 and len(request.form) == 0 and len(request.files) == 0
```

Comment:
I think `not(request.body or request.form or request.files)` would be more readable.

Reply:
yes, it definitely will